### PR TITLE
Changing the background and list styling of further resources

### PR DESCRIPTION
**Pull request by marionb**

### CODE REVIEW

File: thinkhazard/static/less/report.less [hunk 1]
```diff
@@ -223,9 +222,45 @@ body {
...
+    }
+  }
+}
+
+.further-resources-level(@level) {
+  @color: "level-@{level}-color";
+  .level-@{level} {
+    #further-resources li {
+      color: @@color;
+    }
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: You may want to apply this color only to ".fa-li".

File: thinkhazard/static/less/report.less [hunk 2]
```diff
@@ -223,9 +222,45 @@ body {
...
+    li {
+      margin-top: 15px;
+      margin-left: 10px;
+    }
+    a {
+      color: #035e8b;
+    }
+    a:hover {
+      text-decoration: underline;
+    }
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: _color_ and _text-decoration_ for links are to be set the whole application IMO.

**Comment by pgiraud** [comment]
Are you able to make it so the arrows are aligned on the left (ie. below the "F")?

**Comment by marionb** [comment]
I can Align the text with the "F" (by removing the margin from the ul) but I have not relay the arrows. Is there a trick for this?

**Comment by pgiraud** [comment]
Can you show me?

**Comment by pgiraud** [comment]
OK. So let's merge this one and address the alignment later.

**Pull request merged**